modules/claims_api/spec/sidekiq/v2/disability_compensation_docker_container_upload_spec.rb

```diff
@@ -109,6 +109,79 @@
           service.perform(claim.id)
         end.to raise_error(Common::Exceptions::BackendServiceException)
       end
+
+      it "does not retry when the message key is 'in progress' and are in an array" do
+        body = {
+          messages: [{ key: 'form526.InProcess', severity: 'FATAL', text: 'Form 526 is already in-process' }]
+        }
+
+        allow_any_instance_of(ClaimsApi::EVSSService::Base).to(
+          receive(:submit).and_raise(Common::Exceptions::BackendServiceException.new(
+                                       'form526.submit.establshClaim.serviceError', {}, nil, body
+                                     ))
+        )
+        service.perform(claim_with_evss_response.id)
+
+        claim_with_evss_response.reload
+        expect(claim_with_evss_response.status).to eq('errored')
+        expect(claim_with_evss_response.evss_response).to eq([{ 'key' => 'form526.InProcess', 'severity' => 'FATAL',
+                                                                'text' => 'Form 526 is already in-process' }])
+      end
+
+      it 'does retry when the message indicates a birls error and is in an array' do
+        body = {
+          messages: [{ key: 'header.va_eauth_birlsfilenumber', severity: 'ERROR',
+                       text: 'Size must be between 8 and 9' }]
+        }
+
+        allow_any_instance_of(ClaimsApi::EVSSService::Base).to(
+          receive(:submit).and_raise(Common::Exceptions::BackendServiceException.new(
+                                       'form526.submit.establshClaim.serviceError', {}, nil, body
+                                     ))
+        )
+        claim_with_evss_response.reload
+        expect do
+          service.perform(claim.id)
+        end.to raise_error(Common::Exceptions::BackendServiceException)
+        expect(claim_with_evss_response.status).to eq('errored')
+      end
+
+      it 'does retry when the message indicates a birls error and is NOT in an array' do
+        body = { key: 'header.va_eauth_birlsfilenumber', severity: 'ERROR',
```

Comment:
The error I get back from the docker container when manipulating the BIRLS number to be too short is this 
```
[{:key=>"header.va_eauth_birlsfilenumber.Invalid", :severity=>"ERROR", :text=>"Size must be between 8 and 9"}]
```
So it does not have the `messages` key on it but it is an array.  The above is the error exacty as it is when it arrives in the first rescue and sent to get formatted.  So still seeing

<img width="823" alt="Screenshot 2024-04-29 at 12 50 56 PM" src="https://github.com/department-of-veterans-affairs/vets-api/assets/129893414/d2c1a61e-021f-4e6c-95e6-dc917e3439a7">